### ISSUE TITLE
output materials and methods text, pdf?

### ISSUE THREAD

**Issue by pconesa** [I2PC/scipion#365]
With citations to the references.
Something easy for the user to use and add to the future paper.

For the web and for desktop.

**Comment by azazellochg** [comment]
I don't think this is gonna work since summaries/methods are not well written for most of protocols and updating them (#711) first will take years :)  Also this information is by design very scarce and in my opinion will not be used for the papers.
